docs/shared/angular-standalone-tutorial/angular-standalone.md

```diff
@@ -0,0 +1,795 @@
+
+More info [here](/core-features/integrate-with-editors).
+
+{% /callout %}
+
+Run the following command to generate a new "hello-world" component. Note how we append `--dry-run` to first check the output.
+
+```shell {% command="npx nx g @nx/angular:component hello-world --standalone --dry-run" path="myngapp" %}
+>  NX  Generating @nx/angular:component
+
+CREATE src/app/hello-world/hello-world.component.css
+CREATE src/app/hello-world/hello-world.component.html
+CREATE src/app/hello-world/hello-world.component.spec.ts
+CREATE src/app/hello-world/hello-world.component.ts
+
+NOTE: The "dryRun" flag means no changes were made.
+```
+
+As you can see it would generate a new component into the `app/hello-world/` folder. If you want to actually run the generator, just remove the `--dry-run` flag.
+
+```ts {% fileName="src/app/hello-world/hello-world.component.ts" %}
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+
+@Component({
+  selector: 'myngapp-hello-world',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './hello-world.component.html',
+  styleUrls: ['./hello-world.component.css'],
+})
+export class HelloWorldComponent {}
+```
+
+## Building the App for Deployment
+
+If you're ready and want to ship your application, you can build it using
+
+```shell {% command="npx nx build" path="myngapp" %}
+> nx run myngapp:build:production
+
+✔ Browser application bundle generation complete.
+✔ Copying assets complete.
+✔ Index html generation complete.
+
+Initial Chunk Files           | Names         |  Raw Size | Estimated Transfer Size
+main.afa99fe9f64fbdd9.js      | main          | 193.58 kB |                51.57 kB
+polyfills.1acfd3f58d94d542.js | polyfills     |  32.98 kB |                10.66 kB
+runtime.37059233034b21c2.js   | runtime       | 892 bytes |               515 bytes
+styles.ef46db3751d8e999.css   | styles        |   0 bytes |                       -
+
+                              | Initial Total | 227.44 kB |                62.73 kB
+
+Build at: 2023-05-23T14:00:31.981Z - Hash: 9086e92ce0bfefca - Time: 5228ms
+
+ ——————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+
+ >  NX   Successfully ran target build for project myngapp (7s)
+```
+
+All the required files will be placed in the `dist/myngapp` folder and can be deployed to your favorite hosting provider.
+
+## You're ready to go!
+
+In the previous sections you learned about the basics of using Nx, running tasks and navigating an Nx workspace. You're ready to ship features now!
+
+But there's more to learn. You've got two possibilities here:
+
+- [Jump to the next steps section](#next-steps) to find where to go from here or
+- keep reading and learn some more about what makes Nx unique when working with Angular.
```

Comment:
I'm not going to block the PR for this, but I still think the tutorial would be more digestible if it was split into two parts.  Then this section would be:

But there's more to learn:
- Starting where this tutorial left off, [learn to modularize your application and take advantage of the structure Nx provides](/angular-standalone-tutorial/modularize)
- Dig into the underlying [mental model of Nx](/concepts/mental-model)
- Understand the [differences between the Angular CLI and Nx](/concepts/angular-and-nx)
- Learn how to [migrate your existing Angular CLI repo to Nx](/recipes/adopting-nx/migration-angular)